charts/common/values.yaml

```diff
@@ -269,3 +269,10 @@ configmap:
   data: {}
     # MY_CONFIG: yes
 
+secrets:
+# -- add externalSecret to sync secrets from secret manager
+# @default -- {}
```

Comment:
Move above `secrets:` stanza so they get included in doc

Reply:
Moved, and removed the default since it is extrapolated from the actual value.